src/BannerNewsletter/index.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+import { Button, SvgCross } from '@guardian/source-react-components';
+import { useEscapeShortcut, OnCloseClick, CLOSE_BUTTON_ID } from '../bannerCommon/bannerActions';
+import { NewsletterSubscribeCallback, CTA, NewsletterFrequency } from '../newsletterCommon';
+import type { TrackClick } from '../utils/tracking';
+import { styles } from '../styles/bannerCommon';
+import { canRender, COMPONENT_NAME } from './canRender';
+export { COMPONENT_NAME };
+
+export type BrazeMessageProps = {
+    ophanComponentId?: string;
+    header?: string;
+    body?: string;
+    boldText?: string;
+    imageUrl?: string;
+    newsletterId?: string;
+    frequency?: string;
+};
+
+export type Props = {
+    brazeMessageProps: BrazeMessageProps;
+    trackClick: TrackClick;
+    subscribeToNewsletter: NewsletterSubscribeCallback;
+};
+
+const BannerNewsletter: React.FC<Props> = (props: Props) => {
+    const {
+        brazeMessageProps: {
+            ophanComponentId = COMPONENT_NAME,
```

Comment:
Similar to `BannerWithLink`, I don't think we need to provide a fallback/default here as we enforce this as a required param in the `canRender` (and mark it as required for marketing in Storybook).

```suggestion
            ophanComponentId,
```

Reply:
Will remove

Reply:
... Done